jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/ProgressCircular.java

```diff
@@ -31,12 +35,38 @@ public boolean isSpinning() {
        return core().getAttribute("class").contains("v-progress-circular--indeterminate");
     }
 
+    @JDIAction("Gets '{name}' line thickness")
+    public int getThickness() {
+        Matcher mt = Pattern.compile("viewBox=\"(?:[\\d\\.]* ){2}([\\d\\.]*) ([\\d\\.]*)").matcher(core().find(By.cssSelector("svg")).getAttribute("outerHTML"));
+        mt.find();
+        float viewBox = Math.max(Float.parseFloat(mt.group(1)), Float.parseFloat(mt.group(2)));
+        float sizePx = Math.min(Float.parseFloat(width().split("px")[0]), Float.parseFloat(height().split("px")[0]));
+        float strokeWidth = Float.parseFloat(core().find(By.cssSelector("svg circle")).getAttribute("stroke-width"));
+        return Math.round(sizePx*strokeWidth/viewBox);
+    }
```

Comment:
Could you, please, clarify (just for my understanding) why it is calculated in such way. Thanks.

Reply:
Because it calculated this way. There's no straight thickness value in html, it is expressed through these three variables. I do not know why Vuetify team implemented it like this, may be it is connected somehow with autoscaling issues.

Reply:
Thanks for clarification!